packages/common/src/ui/components/popover/PaperPopover/PaperClickPopover/usePaperClickPopover.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { usePopover } from '../../BasePopover';
+import {
+  PaperClickPopover as ClickPopover,
+  PaperClickPopoverProps,
+} from './PaperClickPopover';
+
+export const usePaperClickPopover = () => {
```

Comment:
made the hook to make usage easier. My problem was wanting to use `hide` in both the content and in the popover - so I lifted that out to the consumer, which meant passing it back in, which is annoying.

Reply:
Ah yes I see the problem.. annoying! Good work around though, makes it nice and easy